data/pilots/rebel-alliance/t-65-x-wing.json

```diff
@@ -307,6 +307,262 @@
       "extended": true,
       "keywords": ["X-wing"],
       "epic": true
+    },
+    {
+      "name": "Biggs Darklighter",
+      "caption": "Battle of Yavin",
+      "initiative": 3,
+      "limited": 1,
+      "cost": 5,
+      "standardLoadout": true,
+      "xws": "biggsdarklighter-battleofyavin",
+      "ability": "During the System Phase, you may choose 1 friendly ship at range 1. If you do, treat your initiative as equal to the chosen ship's initiative until the end of the Activation Phase.",
+      "shipAbility": {
+        "name": "Hope",
+        "text": "After another friendly ship at range 0-3 is destroyed, you may perform a [Focus] or [Boost] action."
+      },
+      "upgrades": [
+        {
+          "title": "Attack Speed",
+          "slots": ["Talent"],
+          "ability": "After you fully execute a [3 [Straight]] or [4 [Straight]] maneuver, you may perform a boost using the [1 [Straight]] template. (This is not an action)."
+        },
+        {
+          "title": "Selfless",
```

Comment:
Some of these upgrades already exist, I figured it made more sense to just copy the text here since there's no guarantee in the long term that the text on these cards will match any upgrade. In some cases we know it doesn't, but I'm not sure if there's any non-limited upgrades where that is true.

Reply:
Putting the text of every upgrade on every pilot is too much repetition. Instead, I'd do [what yasb does](https://github.com/raithos/xwing/blob/master/coffeescripts/content/cards-common.coffee).

1: Add new upgrade entries for upgrades which are unique to standardized pilots. This means things like attack run or yavin chewie crew. Upgrades which appear unchanged on standardized loadout pilots should not be duplicated.

Upgrades from standardized loadout cards should:
- have the name of the expansion in the card id (`foo-battleofyavin`)
- have no cost field (although leaving cost as 0 is fine)
- have a new field `standardLoadoutOnly: true`. This should be its own field, it's logically distinct from `restrictions`

This will require updates to the upgrade schema.

2: list upgrade IDs in a pilot's `upgrades` field

Reply:
Came back to make effectively the same comment, that yes this duplication is just too much, even if it means some oddities, like for example if the Selfless on Biggs doesn't get an errata (though I'm not even sure what that would mean).